content/vue/step02.md

```diff
@@ -0,0 +1,124 @@
+{{#template name="vue-step02"}}
+
+# In a hurry?
+
+If you don't have time to follow this detailed tutorial right now, you have two options.
+
+To skip to the finished application explained by this tutorial, run the following commands:
+
+```sh
+git clone git@github.com:meteor/simple-todos-vue.git
+cd simple-todos-vue
+meteor npm install
+meteor
+```
+
+If you are an experienced Vue developer, most of this application will be self-explanatory, though you can always refer back to this tutorial to understand the details.
+
+# Working with Vue
+
+The rest of this tutorial will take you on a step-by-step journey through the details of using Vue with Meteor.
+
+### Install Vue dependencies
+
+To start working with Vue as our view library, we first need to add the relevant npm package.
+
+Open a new terminal in the same directory as your running app, and type:
+
+```sh
+meteor npm install --save vue
+```
+
+> Note: `meteor npm` supports the same features as `npm`, though the difference can be important. Consult the [`meteor npm` documentation](https://docs.meteor.com/commandline.html#meteornpm) for more information.
+
+We also need to add the akryum:vue-component package to allow us to create files with the .vue extension, which makes it possible to create Vue components in the single file format.
+
+```sh
+meteor add akryum:vue-component
+```
+
+### Replace `blaze-html-templates` with `static-html`
+
+By default, new Meteor applications use Blaze as their templating engine. While it's possible for a Meteor application to use Blaze and Vue simultaneously, the application we're building in this tutorial does not need Blaze at all. Since you created your Meteor application with `meteor create` the `blaze-html-templates` package is inlcuded, we recommend that you remove support for Blaze and add support for static HTML templates.
```

Comment:
> Since you created your Meteor application with `meteor create` [...]

This step isn't included in this tutorial earlier. ... and maybe its better going for `meteor create --bare`. In fact, if you use `meteor create --bare` you already have the `static-html` package installed. I'd rather write this step as a check. Or if you aim for providing a tutorial for new projects, advise the user to use `meteor create --bare` to start the project.